filter_block.py

```diff
@@ -33,6 +36,7 @@ class Filter(Block):
             filter.
     """
 
+    version = VersionProperty(version='0.1.0', min_version='0.1.0')
```

Comment:
Let's talk versions too and come up with a block-wide plan for how we want to do them